x-pack/docs/en/watcher/condition/array-compare.asciidoc

```diff
@@ -42,6 +42,9 @@ than or equal to 25:
 <4> The comparison value. Supports date math like the 
     <<compare-condition-date-math, compare condition>>.
 
+:NOTE: When using fieldnames that contain a dot this condition will not
```

Comment:
The leading colon makes the line disappear, at least when I build locally. When I remove it everything looks fine.

Reply:
that one must have been slipped in before committing... thanks for the find! removed it